package.json

```diff
@@ -18,30 +19,59 @@
   },
   "homepage": "https://vaadin.com/components",
   "files": [
-    "vaadin-*.js",
-    "src",
-    "theme"
+    "/vaadin-*.ts",
+    "/vaadin-*.d.ts",
+    "/vaadin-*.d.ts.map",
+    "/vaadin-*.js",
+    "/vaadin-*.js.map",
+    "/src/",
+    "/theme/"
   ],
   "husky": {
     "hooks": {
-      "pre-commit": "npm run lint"
+      "pre-commit": "lint-staged"
     }
   },
+  "lint-staged": {
+    "*.ts": [
+      "eslint --fix",
+      "prettier --write",
+      "git add"
+    ]
+  },
   "scripts": {
-    "test": "wct",
-    "check": "npm-run-all --parallel check:*",
-    "check:bower": "magi check-bower",
+    "analyze": "wca analyze src/vaadin-details.ts --format json --outFile custom-elements.json",
+    "build": "tsc && replace-in-file \"/(import|from)( '[.]+/)(.*)(?<!.js)(';)/g\" \"\\$1\\$2\\$3.js\\$4\" vaadin-*.{js,d.ts},src*.{js,d.ts},theme/**/*.{js,d.ts} --isRegex --verbose",
+    "dist": "rimraf dist && rollup -c rollup.config.js && cp custom-elements.json dist",
```

Comment:
"analyze" should be added here also as the script depends on custom-elements.json

Reply:
Good catch. Also `npm run build` was needed, so I added both.